datasets/us/sec_harmed_investors/us_sec_harmed_investors.yaml

```diff
@@ -1023,3 +1023,8 @@ lookups:
       - match: moravec, matthew
         names:
           - Matthew Moravec
+      - match: sterling watters group lp; sterling watters capital advisors llc; and sterling watters capital management inc.
+        names:
+          - Sterling Watters Group LP
+          - Sterling Watters Capital Advisors LLC
+          - Sterling Watters Capital Management Inc.
```

Comment:
I already did this one :) adjacent to the existing sterling watters entry. I try to colocate them